scripts/apps/legal-archive/views/legal_archive.html

```diff
@@ -1,8 +1,10 @@
 <div class="subnav legal-archive subnav--absolute" sd-media-query min-width="700">
-    <div class="filter-trigger" ng-click="openAdvanceSearch = !openAdvanceSearch"
+    <button class="filter-trigger" ng-click="openAdvanceSearch = !openAdvanceSearch"
+         title="{{ :: 'Filters' | translate }}"
```

Comment:
If it's an icon button and has aria label for accessibility, what is the title for?

Reply:
for sighted users to be able to see th edescription on hover